website/content/api-docs/system/namespaces.mdx

```diff
@@ -94,3 +94,98 @@ $ curl \
   "path": "ns1/"
 }
 ```
+
+## Lock Namespace
```

Comment:
As a consumer of the API docs, I'm presented with how to lock and unlock a namespace. I am not, however, shown how a locked API will respond to requests. There is a **API Locked Response** section in the concepts docs which we might want to link to rather than providing duplicate information that needs to be maintained separately. What do you think?

Reply:
i like that. do you know how i might link that section here?

Reply:
I think it would look something like:

```
[API Locked Response](/concepts/api-locks#api-locked-response)
```

Reply:
8a60e54b0